doc/getting-started/getting-started.rst

```diff
@@ -791,8 +789,9 @@ Launch the User VM
       sudo chmod +x /usr/sbin/iasl
       sudo ~/acrn-work/launch_uos_id3.sh
 
-#. Confirm that you see the console of the User VM on the Service VM's terminal
-   (on the monitor connected to the target system). Example:
+#. It will take a few seconds for the User VM to boot and start running the
+   Ubuntu image.  Confirm that you see the console of the User VM on the Service
+   VM's terminal (on the monitor connected to the target system). Example:
```

Comment:
Recommend removing "on the monitor connected to the target system." Participants 1 and 4 found the wording confusing since they were already looking at the monitor, and my guess is that it's a remnant meant to clarify monitor vs. optional serial console.